### ISSUE TITLE
Disable showing of outputs in `up`

### ISSUE THREAD

**Issue by nesl247** [pulumi/pulumi#5188]
I work with a lot of stacks, and most of them have outputs. However, rarely do I need to see the output when running `pulumi up`. I would expect to only see this when clicking on `details`, but even then, I'd like to not see it.

**Comment by justinvp** [comment]
Workaround would be to specify:

```
pulumi up --suppress-outputs
```

**Comment by nesl247** [comment]
Yep, but having to specify that all the time is a little annoying. I think one of the things that would benefit pulumi is if it had a difference between a "show the user this" type of output, and a "this is output for a `StackReference`". 100% of our outputs are for `StackReference` purposes. We NEVER need to see these.

**Comment by stack72** [comment]
Would setting an environment variable to suppress these be useful for you @nesl247 ?

**Comment by nesl247** [comment]
That works. I don't think it's a good long term solution based on the separation of different kinds of outputs, but it works for now.

**Comment by ltrojan-ths** [comment]
any update on this?

it's very annoying, especially when errors are emitted and one has pages of unnecessary outputs to scroll back before finding he interesting bits...

**Comment by danktec** [comment]
Sometimes a resource may have a sensitive string as part of its output/diff. I'd like to optionally suppress or redact the output of a given resource so it's not echoing secrets all over the place.

**Comment by enmanuelr** [comment]
Some resources, like cloudflare workers or AWS lambda functions, can produce very large outputs. This can be difficult to work with at best, and can cause stability issues in a terminal session at work.

It would be fantastic if we had an optional setting we could pass to `CustomResourceOptions`, to omit the output of a specific resource.

**Comment by youroff** [comment]
Would like to bump this issue: kubernetes clusters are extremely noisy and we cannot avoid exporting them. Every time end up having to scroll several screens only to see what's going to be updated when doing `pulumi up`. Ideally, it should be an option in Pulumi.yaml, and for the sake of uniformity, it could a default state for any supported flag. Perhaps, something like this?

```
name: Example Pulumi project file with only required attributes
runtime: nodejs
cli:
    suppress-outputs: true
```
